Install failing: Command failed: /usr/local/bin/git checkout 5f0b3799f50944bc4d1eb58acdef61c38883d6cf

npm install is failing due to a git dependency. It's weird because I see the commit here: https://github.com/LearnBoost/websocket.io/commit/5f0b3799f50944bc4d1eb58acdef61c38883d6cf

```
$ npm install markserv -g
npm ERR! code 128
npm ERR! Command failed: /usr/local/bin/git checkout 5f0b3799f50944bc4d1eb58acdef61c38883d6cf
npm ERR! fatal: reference is not a tree: 5f0b3799f50944bc4d1eb58acdef61c38883d6cf
npm ERR!

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/dan/.npm/_logs/2017-07-24T16_18_09_255Z-debug.log
```

Anyone else getting the same issue?

--- a/package.json
+++ b/package.json
@@ -36,7 +36,7 @@
     "open": "0.0.5",
     "openport": "0.0.4",
     "send": "^0.14.1",
-    "websocket.io": "git://github.com/LearnBoost/websocket.io#5f0b3799f50944bc4d1eb58acdef61c38883d6cf"
+    "websocket.io": "git://github.com/kewisch/websocket.io#require-events"
   },
   "devDependencies": {
     "connect-livereload": "^0.6.0",

